### ISSUE TITLE
Global APT update (package update, automatic package update) causes memory to leak and eventual crash

### ISSUE THREAD

**Issue by RoldyBuffalo** [whiskerz007/proxmox_hassos_install#78]
I have been running this install for weeks, I've had this memory leak in proxmox since I first installed it, wondering if there is anyway I can disable automatic apt updates (this is done by cron I believe) as every time it updates the package list, my hassos vm's memory explodes causing several issues. this is just the apt update doing this, not even an upgrade.

**Comment by orditeck** [comment]
I feel like I might have the same issue. My VM keep crashing at specific time, eg 8:00.

**Comment by RoldyBuffalo** [comment]
Woah, I have to say. I really, really truly though I was alone. Let me check my current build and see if it persisted. I FULLY rebuilt the stack, reflashed the media with proxmox, reinstalled hassio (home assistant) restored snapshot, and it persisted. I wiped away home assistant, then fully rebuilt home assistant. It seems to me that it has something to do with the dns of the hassos instance. I was setting up network wide dns on the hassos instance, and THAT was what was screwing it up. FUTHERMORE, uninstalling the dns, be it pi-hold or whatever, DID NOT SOLVE THE ISSUE... I had to rebuild the stack once more. New, fresh install of hassos, set EVERYTHING up how it was. (You can even have both instances open at once if you're tricky and change the port on one) move over your config files, BUT DO NOT TOUCH ANYTHING DNS RELATED LIKE PI-HOLE OR ADGUARD. 

You should be fine. 

You have know, you will go down this rabbit hole, and you may never come out of it. I nixxed alot of my auxillierary smart home stuff, just too much of a head ache. and after @whiskerz007 (knows) 9 months if it doing exactly as you and I have witnessed, and for me personally, zero other souls alive to help, or that could help....

I did what I had to do. @orditeck

**Comment by whiskerz007** [comment]
Is Proxmox DNS configured to use the Home Assistant VM addon?

**Comment by orditeck** [comment]
In my case it seemed very, very specific to my use case. I still hardly believe it but I don't see anything else and it does make sense.

I have CC2531 plugged to the host. The VM use this device for zigbee2mqtt. **The USB device was in a metal electric box**.

My only current hypothesis is that I know the USB dongle sparked at least once on the electric box, I saw it. I think that the host was dropping the USB connection to protect itself, leading in the VM to be hanging for a return from the host, leading it to become unresponsive.

I moved the dongle outside the box after I wrote here and never got an issue after that...
